src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -39,6 +39,7 @@ public abstract class WorkspaceModel : NotificationObject, ILocatable, IUndoRedo
         private readonly ObservableCollection<NodeModel> nodes;
         private readonly ObservableCollection<NoteModel> notes;
         private readonly UndoRedoRecorder undoRecorder;
+        private Guid guid;
```

Comment:
I really don't like relying on the default struct initilizer code for Guids. I'm not even sure if it's Guid.Empty - in which case this won't work.

Could we add a call in the ctor of the model guid = Guid.NewGuid() or something similar?